source/openstack.rst

```diff
@@ -1,3 +1,6 @@
+.. toctree::
+.. sectnum::
+
```

Comment:
Should we add this to the opennebula.rst?

Reply:
will do.